Amazon Alexa Reviews Analysis/Model/prediction.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+import Amazon_Alexa_Reviews_Analysis as A

```

Comment:
as the file will be renamed, update it here also. Also, if you use an alias for this then use a better word.